automation/run-tests.sh

```diff
@@ -283,8 +215,40 @@ function upload_coverage {
     fi
 }
 
+function check_iface_exist {
+    exec_cmd "ip a s | grep -q $1"
+}
+
+function prepare_network_environment {
+    set +e
+    check_iface_exist 'eth1'
```

Comment:
It would be nicer to use a loop here:

```
for device in eth1 eth2;
do
    if !check_iface_exist $device
         CREATED_DEVICES = "${CREATED_DEVICES} $device
...
```

This also allows to move "eth1 eth2" into a variable and simplifies changing them in the future.

Reply:
I agree. Will do. Thanks!

Reply:
Done, thanks!